flutter/Dockerfile

```
@@ -1,5 +1,10 @@
 FROM circleci/android:api-30
 
+RUN sudo apt-get install -y locales
+RUN sudo dpkg-reconfigure locales
+RUN /usr/sbin/locale-gen C.UTF-8
+ENV LANG='C.UTF-8' LC_ALL='C.UTF-8'
```

Should this be:

```suggestion
set LANG='C.UTF-8' LC_ALL='C.UTF-8'
```

I received the following when manually entering the commands:

```
circleci@de89be5839c0:~/workforce/android$ ENV LANG='C.UTF-8' LC_ALL='C.UTF-8'
-bash: ENV: command not found
```

---

`ENV` is a docker command, like `RUN`. It isn't a BASH command.